src/test/java/io/github/linuxforhealth/hl7/message/Hl7ORUMessageTest.java

```diff
@@ -239,18 +234,22 @@ public void multipleOBXWithNoId() throws IOException {
       Assert.assertTrue("Incorrect content type", a.getContentType().equalsIgnoreCase("text"));
       Assert.assertTrue("Incorrect language", a.getLanguage().equalsIgnoreCase("en"));
       
-      //TODO: Validate data by decoding
-      //Assert.assertTrue("Incorrect data", Base64.getDecoder().decode((a.getData())).equals("~[PII] Emergency Department~ED Encounter Arrival Date: [ADDRESS] [PERSONALNAME]:~"));
-      //Base64.getEncoder().encodeToString(val.getBytes());
-      
+      //Verify data attachment after decoding
+      String decoded = new String(Base64.getDecoder().decode(a.getDataElement().getValueAsString()));
+      Assert.assertTrue("Incorrect data", decoded.equals("~[PII] Emergency Department~ED Encounter Arrival Date: [ADDRESS] [PERSONALNAME]:~"));
+
       Assert.assertTrue("Incorrect title", a.getTitle().equalsIgnoreCase("ECHO CARDIOGRAM COMPLETE"));
 
-      //TODO: validate date 
-//      System.out.println(OffsetDateTime.of(2020, 8, 2, 12, 44, 55, 0, ZoneOffset.of("+08:00")));
-//      Assert.assertTrue("Incorrect creation date", a.getCreation().equals("2020-08-02T12.44:55+08.00"));
+      //Verify creation data is persisted correctly - 2020-08-02T12:44:55+08:00
+      Calendar c = Calendar.getInstance();
+      c.clear();
```

Comment:
1. Is clear() needed?
2. Remove extra space in the set() call
3. My ORU-multiline-short.hl7 has 20200802124455 -- just double checking this is right, your file must be different, perhaps I do not have the latest
4. Not sure about the timezone and how that is set, is it going to work universally (in the build, etc)